packages/eslint-plugin/package.json

```diff
@@ -33,6 +33,7 @@
 		"eslint-plugin-react": "^7.14.3",
 		"eslint-plugin-react-hooks": "^1.6.1",
 		"globals": "^12.0.0",
+		"prettier": "npm:wp-prettier@1.19.1",
```

Comment:
This matches what was used in #18048 for `wp-scripts` in d20fa4513194ff1ecab7bf8493162cad50ed2332.